src/server/pfs/drive/driver.go

```diff
@@ -1053,8 +1054,8 @@ func (r *fileReader) Read(data []byte) (int, error) {
 			return 0, io.EOF
 		}
 		var err error
-		r.reader, err = pfsclient.GetBlock(r.blockClient,
-			r.blockRef().Block.Hash, uint64(r.offset), uint64(r.size))
+		client := client.APIClient{BlockAPIClient: r.blockClient}
```

Comment:
Ah, this is the trade we make for the consolidated interface for users. Not a bad trade.